test_unstructured/documents/structured_jsons/example.json

```diff
@@ -0,0 +1,121 @@
+        "metadata": {
+            "category_depth": 3,
+            "page_number": 1,
+            "parent_id": "45b3d005-3468-484b-a1c7-b53998115412",
+            "text_as_html": "<h1 class=\"Title\" id=\"c95473e8-a370-4fc2-b418-697f9fddb27b\">Header </h1>"
+        },
+        "text": "Header",
+        "type": "Title"
+    },
+    {
+        "element_id": "379cbfdc-16d4-4bd6-a59e-6cfabe6438d5",
+        "metadata": {
+            "category_depth": 3,
+            "page_number": 1,
+            "parent_id": "45b3d005-3468-484b-a1c7-b53998115412",
+            "text_as_html": "<time class=\"CalendarDate\" id=\"379cbfdc-16d4-4bd6-a59e-6cfabe6438d5\">Date: October 30, 2023 </time>"
+        },
+        "text": "Date: October 30, 2023",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "637c2f69-35fb-4353-a5f7-3025ce04619d",
+        "metadata": {
+            "category_depth": 2,
+            "page_number": 1,
+            "parent_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+            "text_as_html": "<form class=\"Form\" id=\"637c2f69-35fb-4353-a5f7-3025ce04619d\"> <label class=\"FormField\" for=\"company-name\" id=\"50027ccc-be19-48c9-853c-e0de37b635c2\">From field name </label><input class=\"FormFieldValue\" id=\"company-name\" value=\"Example value\" /></form>"
+        },
+        "text": "From field name",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "59242237-3ed7-41b6-8a07-7e2003f8ed81",
+        "metadata": {
+            "category_depth": 2,
+            "page_number": 1,
+            "parent_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+            "text_as_html": "<section class=\"Section\" id=\"59242237-3ed7-41b6-8a07-7e2003f8ed81\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "dc3792d4-422e-444f-9087-6b56d0cfb20d",
+        "metadata": {
+            "category_depth": 3,
+            "page_number": 1,
+            "parent_id": "59242237-3ed7-41b6-8a07-7e2003f8ed81",
+            "text_as_html": "<table class=\"Table\" id=\"dc3792d4-422e-444f-9087-6b56d0cfb20d\"> <thead class=\"TableHeader\" id=\"50a5548a-87e8-4024-af59-0b3d2830d140\"> <tr class=\"TableRow\" id=\"5e473d77-4247-4412-be72-dc4e2c45bd4a\"> <th class=\"TableCellHeader\" id=\"01800309-aa42-411c-98ae-30f85b23f399\">Description </th><th class=\"TableCellHeader\" id=\"c2765b63-d089-46a2-8519-55e79e301de4\">Row header </th></tr></thead><tbody class=\"TableBody\" id=\"e0a9a8ff-dd71-48ad-8b4a-274b073d340a\"> <tr class=\"TableRow\" id=\"77e82997-4632-4551-9133-0b0b8545d1e3\"> <td class=\"TableCell\" id=\"7fee12d4-c555-4b7d-a778-d6f8fdec8a57\">Value description </td><td class=\"TableCell\" id=\"5a7a33b0-c57b-4eb8-81a3-5bce9f87c831\"> <span class=\"Currency\" id=\"87220f9d-62c3-482e-92e7-de72a26869cd\">50 $ </span><span class=\"Measurement\" id=\"0095b9ef-b90a-4cca-991e-73547c7165f1\">(1.32 %) </span></td></tr></tbody></table>"
```

Comment:
**non-blocking comment for this PR**, something to follow up on later: these `id`'s and `class`s per cell is adding a lot of bloat to the table. it should be safe to drop `id`s, maybe also `class`s? (for consistency with other table `text_as_html`